webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/pdf/util/PdfUtil.kt

```diff
@@ -152,22 +146,6 @@ object PdfUtil {
         }
     }
 
-    tailrec fun binarySearchInc(min: Float, max: Float, precision: Float, shouldIncrease: (Float) -> Boolean): Float {
```

Comment:
I don't think this should be removed just because it is currently not used. You yourself pointed out how much troble incorrect bifurcation can cause with binary search and I'd prefer for this to "lay around unused" just in case we might ever need it.

Reply:
Ok